modules/social_features/social_group/social_group.install

```diff
@@ -544,3 +545,17 @@ function social_group_update_13000() : void {
 function social_group_update_13002() : void {
   \Drupal::state()->delete("social_group_group_type_migration_opt_out");
 }
+
+/**
+ * Add a medium teaser view mode.
+ */
+function social_group_update_13003(): void {
+  $config_path = \Drupal::service('extension.list.module')->getPath('social_group') . '/config/install';
```

Comment:
Please make sure to use update_helper for this hook update.

Reply:
Done